modules/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3Repository.java

```diff
@@ -197,6 +198,21 @@ class S3Repository extends MeteredBlobStoreRepository {
         Setting.Property.Dynamic
     );
 
+    /**
+     * We will retry deletes that fail due to throttling. We use an {@link BackoffPolicy#exponentialBackoff(TimeValue, int)}
+     * with the following <code>initialDelay</code> and <code>maxNumberOfRetries</code>
+     */
+    static final Setting<TimeValue> RETRY_THROTTLED_DELETE_INITIAL_DELAY = Setting.timeSetting(
+        "throttled_delete_retry.initial_delay",
+        new TimeValue(50, TimeUnit.MILLISECONDS),
+        new TimeValue(50, TimeUnit.MILLISECONDS)
+    );
+    static final Setting<Integer> RETRY_THROTTLED_DELETE_MAX_NUMBER_OF_RETRIES = Setting.intSetting(
+        "throttled_delete_retry.max_number_of_retries",
+        8,
+        0
+    );
```

Comment:
Are we OK with these as defaults, it's rather short, they're the same defaults as `BackoffPolicy.exponentialBackoff()` which states:

```
     * Creates an new exponential backoff policy with a default configuration of 50 ms initial wait period and 8 retries taking
     * roughly 5.1 seconds in total.
```